workers/unity/Packages/com.improbable.gdk.core/CodegenAdapters/ComponentDispatcherHandler.cs

```diff
@@ -41,11 +42,30 @@ protected ComponentDispatcherHandler(WorkerSystem worker, World world)
             Worker = worker;
         }
 
+        /// <summary>
+        ///     A helper method that wraps TryGetEntity on the Worker. This throws if the entity does not exist.
+        ///     The reason is throws is that this method is called on receiving ops and not having that Entity Id is
```

Comment:
`the reason is throws is`-> `the reason it throws is`

Reply:
Also "*if* this method".
Can actually remove the first part of the sentence. "This throws if the entity does not exist; calling this method on receive ops ..."

Reply:
Sure will readjust the wording :)